cli/reportgen.sh

```diff
@@ -1,42 +1,30 @@
 #!/bin/bash
 
-cd tests/contract-playground
-forge build --ast
-FOUNDRY_PROFILE=sample forge build --ast
-cd ../../
-
-export ADERYN_SKIP_BUILD=1
-
 #### MARKDOWN REPORTS ######
 
 # Basic report.md 
-cargo run -- ./tests/contract-playground &
-
-# Basic report.profile.md 
-FOUNDRY_PROFILE=sample cargo run -- -o report.sample_profile.md ./tests/contract-playground &
+cargo run --  --scope src/ --exclude lib/ ./tests/contract-playground &
 
 # Create report-config.md based on config file
-cargo run -- --config-file ./tests/aderyn.config.json -o report-config.md ./tests/contract-playground/  &
+cargo run -- --config-file ./tests/aderyn.config.json --exclude lib/ -o report-config.md ./tests/contract-playground/  &
 
 # Create report.judge.md 
-cargo run -- ./tests/contract-playground -o judgeops/current/report.judge.md &
+cargo run --  --scope src/ --exclude lib/ -o judgeops/current/report.judge.md  ./tests/contract-playground &
```

Comment:
Although we don't rely on frameworks, I think we must take clues for scope and exclude from famous files `foundry.toml`